### PR TITLE
Add Helper.is_windows?

### PR DESCRIPTION
**Pull request by janpio**
Adds a convenience method to check if we are on Windows

### CODE REVIEW

**Review by hjanuschka — APPROVED** [review]
🏠

**Review by KrauseFx — APPROVED** [review]
👍

File: fastlane_core/lib/fastlane_core/helper.rb [hunk 1]
```diff
@@ -120,6 +120,10 @@ def self.is_mac?
       self.mac?
     end
 
+    def self.is_windows?
+      self.windows?
```
Comment (giginet) [inline fastlane_core/lib/fastlane_core/helper.rb]: you can also use `alias_method`

**Review by giginet — APPROVED** [review]

File: fastlane_core/lib/fastlane_core/helper.rb (hunk 1 above)
Comment (KrauseFx) [reply fastlane_core/lib/fastlane_core/helper.rb]: Oh, great idea, and we could use the same for other methods also, like `is_mac?`, etc., right? :)

**Pull request merged**